osu.Game/Beatmaps/Formats/OsuLegacyDecoder.cs

```diff
@@ -56,36 +59,37 @@ private enum Section
             TimingPoints,
             Colours,
             HitObjects,
+            Variables,
         }
 
-        private void handleGeneral(Beatmap beatmap, string key, string val)

```

Comment:
even github isn't smart enough to realise it's not a keyword here:

![allow_for_variables_in_storyboards_by_coldvolcano_ _pull_request__866_ _ppy_osu](https://cloud.githubusercontent.com/assets/191335/26581430/647f038c-4577-11e7-921c-4a29382f0c5b.png)